applications/graph/MeshGraphNet/GNNComponents.py

```diff
@@ -0,0 +1,240 @@
+import lbann
+from lbann.modules import Module, ChannelwiseFullyConnectedModule
+
+
+class MLP(Module):
+  """
+    Applies channelwise MLP with ReLU activation with Layer Normalization 
+    with a specified number of hidden layers
+  """
+  global_count = 0
+
+  def __init__(self,
+               in_dim,
+               out_dim,
+               hidden_dim,
+               hidden_layers,
+               norm_type=lbann.LayerNorm,
+               name=None):
+
+    super().__init__()
+    MLP.global_count += 1
```

Comment:
do we have to give names to MLPs?

Reply:
I usually add it for compute graph visualization and for better debugging messages when things break